components/builder-api-client/src/builder.rs

```diff
@@ -499,15 +508,18 @@ impl BuilderAPIProvider for BuilderAPIClient {
     ///
     /// * Remote Builder is not available
     fn delete_origin_secret(&self, origin: &str, token: &str, key: &str) -> Result<()> {
+        debug!("Deleting origin secret: {}, {}", origin, key);
```

Comment:
Same suggestion regarding the name "key".

Reply:
OK